src/modules/75-cd/components/Environments/EnvironmentTabs.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { TabNavigation } from '@harness/uicore'
+import { useStrings } from 'framework/strings'
+
+import routes from '@common/RouteDefinitions'
+import type { ModulePathParams, ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import { useFeatureFlag } from '@common/hooks/useFeatureFlag'
+import { FeatureFlag } from '@common/featureFlags'
+
+export default function EnvironmentTabs() {
+  const { getString } = useStrings()
+  const { accountId, orgIdentifier, projectIdentifier, module } = useParams<ProjectPathProps & ModulePathParams>()
+  const isEnvGroupEnabled = useFeatureFlag(FeatureFlag.ENV_GROUP)
+  const [navLinks, setNavLinks] = useState([])
+
+  useEffect(() => {
+    const links: any = [
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232281)
```suggestion
    const links: unknown = [
```